boards/shields/nrf700x_nrf54h20dk/nrf700x_nrf54h20dk.overlay

```diff
@@ -0,0 +1,173 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <freq.h>
+
+/* Wi-Fi needs more flash (1M) and RAM (512K):
+ *
+ * To minimize the changes, only necessary nodes are added, so, delete
+ * any linked nodes. This was done here as it's applicable to most samples,
+ * if a sample needs anything extra, it can override this with a sample specific
+ * overlay.
+ */
+ /delete-node/ &ipc0;
+ /delete-node/ &cpuppr_vpr;
+ /delete-node/ &mram1x;
+ /delete-node/ &cpurad_uicr;
```

Comment:
why are we deleting stuff like ppr vpr node here?

Reply:
it's not needed for Wi-Fi.